pyEpiabm/pyEpiabm/sweep/initial_infected_sweep.py

```diff
@@ -104,6 +104,7 @@ def __call__(self, sim_params: dict):
                                            ["initial_infected_number"]))
         for person in pers_to_infect:
             person.update_status(InfectionStatus.InfectMild)
+            person.household.remove_susceptible_person(person)
```

Comment:
ERROR 1: Before adding this line, we had errors in basic_simulation.py in which some individuals who started with status 4 (InfectMild) changed to status 2 (Exposed). This error was caused by one infected person infecting another infected person, as they were not removed from the susceptible list of their household in the initial_infected_sweep. Line 107 above is the fix.

Reply:
That makes sense - very good spot thank you!